coverage/cmdline.py

```diff
@@ -755,6 +788,7 @@ def unglob_args(args):
             report      Report coverage stats on modules.
             run         Run a Python program and measure code execution.
             xml         Create an XML report of coverage results.
+            json        Create a JSON report of coverage results.
```

Comment:
This should go between "html" and "report" (alphabetically)

Reply:
addressed